### PR TITLE
Allow specifying branch to deploy

### PR DESCRIPTION
**Pull request by ghost**
See also:
https://github.com/caktus/margarita/pull/127
https://github.com/caktus/developer-documentation/pull/44

### CODE REVIEW

**Comment by rebecca-caktus** [comment]
Updated based on experience in Hosted TracPro: https://github.com/caktus/hosted-tracpro/pull/20

File: conf/pillar/staging.sls [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 environment: staging
 
+branch: master
```
Comment (vkurup) [inline conf/pillar/staging.sls]: I'd probably point this at `develop` to match our usual workflow.

**Comment by vkurup** [comment]
I think this is a useful change. I've updated the branch and changed the staging deploy to use the `develop` branch as a default.

**Comment by dpoirier** [comment]
:deciduous_tree:

**Pull request merged**